### ISSUE TITLE
[BUG] Install routine does not check the correct expose of ./p folder

### ISSUE THREAD

**Issue by math-GH** [FreshRSS/FreshRSS#3679]
**Describe the bug**

If the domain does not point directly to ./p folder it is possible to install FreshRSS without any notice in step1 in the install routine.

![grafik](https://user-images.githubusercontent.com/1645099/123684021-3cdb0480-d84d-11eb-872c-0709621ac082.png)

In the manual it is described:
![grafik](https://user-images.githubusercontent.com/1645099/123684248-87f51780-d84d-11eb-996c-6878c750938b.png)

If the ./p folder is not correct exposed the other folders (f.e. ./data folder) are online available, where imporant private information are stored.  They are (mostly) protected with .htaccess (only valid for Apache server)

**To Reproduce**
1. fresh installation
2l set domain to root, not to /p
3. follow the installation

**Expected behavior**
red/yellow notification in step 1, that the domain is not correct set.

**Environment information (please complete the following information):**
 - FreshRSS version: 1.18.1

**Additional context**
that was the reason for #3657

**Comment by Frenzie** [comment]
Implementation note: this falls under recommended like zlib and mbstring. This is not an error like permissions.

**Comment by Alkarex** [comment]
Indeed, while `/some/path/FreshRSS/p/...` makes sense in some situations, it is quite likely that `/p/...` is an installation error and we should probably warn about it

**Comment by Elgeryy1** [comment]
I'll take this one. Planning to add a non-blocking check in installation step 1 that compares `DOCUMENT_ROOT` against the `./p` folder and warns when they don't match, similar to the existing file/permission checks. Opening a PR shortly.
